extension/tpch/dbgen/bm_utils.cpp

```diff
@@ -141,13 +139,13 @@ long yes_no(char *prompt) {
  * and using the characters in alphanum (currently includes a space
  * and comma)
  */
-void tpch_a_rnd(int min, int max, int column, char *dest) {
```

Comment:
instead of passing the seed idx, passing a pointer to the seed_t object directly.